gramhopper/responses/preset_responses.py

```diff
@@ -10,11 +11,16 @@ class _PresetTextResponse(BaseResponse):
     def __init__(self, preset_response: Union[str, List[str]]):
         self.preset_responses = preset_response
 
+    @abc.abstractmethod
+    def respond(self, bot: Bot, update: Update, response_payload: dict) -> None:
+        pass
+
     def get_response_text(self):
         if isinstance(self.preset_responses, str):
             return self.preset_responses
-        elif isinstance(self.preset_responses, list):
+        if isinstance(self.preset_responses, list):
```

Comment:
Why is this better than elif?

Reply:
Because you don't need `else` after `return`.

Reply:
k